### ISSUE TITLE
Custom tiles

### ISSUE THREAD

**Issue by skairunner** [blind-coder/pz-mapmap#4]
Hi, I'm attempting to render the Daegu map which uses custom tiles that aren't .pack but rather loose, in the form of pngs: 

![image](https://user-images.githubusercontent.com/3517115/140614342-0d7516b6-46f4-4731-b1dc-e2431ba8c3db.png)

The result images unfortunately seem to be missing the custom tiles:

![image](https://user-images.githubusercontent.com/3517115/140614434-33a1cbda-0ca2-40f6-bb1e-fe06869cfc38.png)

Is there a good way to reference the custom tiles? It seems that there are _some_ .pack files, would I be able to just add those with the `-gfxsource` argument and have it work?

**Comment by blind-coder** [comment]
Yes, you can add any number of .pack files with the -gfxsource parameter and they'll be recognized.

Am 6. November 2021 16:11:52 MEZ schrieb Skye Im ***@***.***>:
>Hi, I'm attempting to render the Daegu map which uses custom tiles that aren't .pack but rather loose, in the form of pngs: 
>
>![image](https://user-images.githubusercontent.com/3517115/140614342-0d7516b6-46f4-4731-b1dc-e2431ba8c3db.png)
>
>The result images unfortunately seem to be missing the custom tiles:
>
>![image](https://user-images.githubusercontent.com/3517115/140614434-33a1cbda-0ca2-40f6-bb1e-fe06869cfc38.png)
>
>Is there a good way to reference the custom tiles? It seems that there are _some_ .pack files, would I be able to just add those with the `-gfxsource` argument and have it work? 
>
>-- 
>You are receiving this because you are subscribed to this thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/blind-coder/pz-mapmap/issues/4
-- 
Diese Nachricht wurde von meinem Android-Gerät mit K-9 Mail gesendet.

**Comment by skairunner** [comment]
But no way to register a loose png file, right?

I'll run the command with the daegu packs and hope it works out. Thanks for
the prompt reply!

**Comment by blind-coder** [comment]
No, individual png files are unfortunately not supported at the moment. Is that common among map creators?

**Comment by skairunner** [comment]
I don't actually know, I'm not a mapmaker myself. I think maps with custom
assets are relatively rare in general
